submodules/eks/iam.tf

```diff
@@ -19,6 +19,31 @@ data "aws_iam_policy_document" "eks_cluster" {
   }
 }
 
+data "aws_iam_policy_document" "load_balancer_controller" {
+  statement {
+    sid     = ""
+    effect  = "Allow"
+    actions = ["sts:AssumeRoleWithWebIdentity"]
+
+    condition {
+      test     = "StringEquals"
+      variable = "${replace(aws_iam_openid_connect_provider.oidc_provider.id, "/arn:.*:oidc-provider//", "")}:sub"
+      values   = ["system:serviceaccount:domino-platform:${var.deploy_id}-load-balancer-controller"]
```

Comment:
The hardcoded namespace is fine for us internally, but might pose a problem for anyone trying to use this off-the-shelf

Reply:
you are right, I checked and there is no namespace parameter being injected in the TF, should I add the parameter or it is not intended? because from infra perspective having the namespace it could be misleading because the people running the TF could be a different that the one installing Domino although in this case they should provide the same value.

to be honest not 100% sure if this should be living here but as far as I understand this is the only available place at least right now.

What should we do?

Reply:
@miguelhar @fraenkel any thoughts on how we might want to handle this in the future and how it could be handled here?

Reply:
Both AKS and GKE pass a map of namespace names which is required.
The more I think about this, the more I think the management of the workload identities would best be served by the agent. Pushing it to the TF requires one to always make sure you are using the TF of the domino version so you can pick up any changes that may occur for a given release. I hit this hard when we moved hephaestus to the compute namespace. I don't want to get into the complications of doing so since it pushes additional requirements when running the agent.